### DISCUSSION TITLE
Pseudonyme

### DISCUSSION THREAD

**Discussion by jaalmoes** [beng-git/DARC2022#7] [Q&A] [answered]
Bonjour,

Quans vous dites dans le diaporama " Toutes les méthodes doivent utiliser un pseudonyme (un entier)" . Je ne comprend pas ce qu'est le pseudonyme dans ce contexte. S'agit-il de la première colonne de la bdd ?

**Comment by nancy-awad** [comment, marked as answer]
Un pseudonyme dans notre contexte est la premiere colonne de la bdd, qui sert à identifier les records par individu.
Dans le fichier anonymisé, vous devez changer les pseudonymes des individus tout en respectant 2 règles :
1)	Dans une semaine un individu a exactement 1 seul pseudonyme.  Vous pouvez changer le pseudonyme d’un même individu d’une semaine à une autre.
2)	Pour les records que vous voulez supprimer, la valeur dans cette colonne doit être ‘DEL’

**Reply by jaalmoes** [reply]
Merci beaucoup
